README.adoc

```diff
@@ -168,11 +168,11 @@ server.xml
 include::finish/src/main/liberty/config/server.xml[]
 ----
 
-After you replace the `server.xml` file, Open Liberty automatically reloads its configuration. 
+After you replace the `server.xml` file, Open Liberty automatically reloads its configuration.
 
-The features that are required to create the CDI producer for `MongoDB` are `cdi-2.0`, `ssl-1.0`, `mpConfig-1.4`, and `passwordUtilities-1.0`, which are all specified in the [hotspot=featureManager file=0]`featureManager` element. The Secure Sockets Layer (SSL) context is configured in the [hotspot file=0]`server.xml` file so that the application can connect to `MongoDB` with TLS. The [hotspot=keyStore file=0]`keyStore` element is configured to point to the `truststore.p12` keystore file that was created in one of the earlier sections. The [hotspot=ssl file=0]`ssl` element is configured to use the `defaultKeyStore` as the keystore and `outboundTrustStore` as the truststore.
+The features that are required to create the CDI producer for MongoDB are `cdi-2.0`, `ssl-1.0`, `mpConfig-1.4`, and `passwordUtilities-1.0`, which are all specified in the [hotspot=featureManager file=0]`featureManager` element. The Secure Sockets Layer (SSL) context is configured in the [hotspot file=0]`server.xml` file so that the application can connect to MongoDB with TLS. The [hotspot=keyStore file=0]`keyStore` element is configured to point to the `truststore.p12` keystore file that was created in one of the earlier sections. The [hotspot=ssl file=0]`ssl` element is configured to use the `defaultKeyStore` as the keystore and `outboundTrustStore` as the truststore.
```

Comment:
Should break up paragraph into multiple lines

Reply:
Will check with ID